app/models/saved_claim.rb

```diff
@@ -32,6 +32,11 @@ class SavedClaim < ActiveRecord::Base
     self.form_id = self.class::FORM.upcase
   end
 
+  def self.add_form_and_validation(form_id)
+    const_set('FORM', form_id)
```

Comment:
this seems a little too meta, why do you need to dynamically set a constant?

Reply:
the form_id attribute is set based on the `FORM` constant. i also wanted to add a validation on form_id that it is only equal to the `FORM` constant